packages/suite/src/support/messages.ts

```diff
@@ -4917,7 +4917,7 @@ export default defineMessages({
         id: 'AMOUNT_IS_TOO_LOW',
     },
     AMOUNT_IS_BELOW_DUST: {
-        defaultMessage: 'Amount is below the dust limit ({dust})',
+        defaultMessage: 'Amount must be greater than the dust limit ({dust})',
```

Comment:
This change would be overwritten by next Crowdin synchronization, see https://satoshilabs.slack.com/archives/CKZHV2G13/p1662039720132549

I just applied this one for you https://crowdin.com/translate/trezor-suite/103/en-en?filter=basic&value=3#q=AMOUNT_IS_BELOW_DUST